src/site/content/en/blog/web-share-target/index.md

```diff
@@ -137,7 +139,9 @@ and that `enctype` be present. Additionally, `enctype` must be
 You must also add a `files` array defining the types of files your app accepts. The
 array elements are entries with two members: a `name` field and an `accept`
 field. The `accept` field takes a MIME type, a file extension, or an array
-containing both.
+containing both. It's best to provide an array, that includes both a
+MIME type, and a file extension, since operation systems differ in which
+they prefer.
```

Comment:
```suggestion
containing both. It's best to provide an array that includes both a
MIME type and a file extension since operating systems differ in which
they prefer.
```

Reply:
Don't use commas when the subordinate clauses are restrictive, meaning they're essential to the meaning of the sentence.